andela/travel_tool_front
# #161595806 view uploaded documents

#### What does this PR do?
This PR enables the user to view uploaded documents

#### Description of Task to be completed?
- Add tests scripts for new feature
- Add components, actions, sagas

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory -` cd travel_tool_front`
- Pull and checkout into this branch - `ft-view-uploaded-documents-161595806`
- Install project dependencies using `yarn install`
- Clone the [Travela backend](https://github.com/andela/travel_tool_back)
- Run database migrations using `yarn run db:rollmigrate`
- Check `.env.example` and set up your `.env` appropriately
- Start the Server and Frontend with `make start` and log into the application

- Note: To create documents for yourself
Replace the **userId** column of the documents on the **Documents** table,  with your **userId** from the **Users** table 

- When logged into Travela, navigate to **Documents** Tab
- You should see your documents 

#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#161595806](https://www.pivotaltracker.com/story/show/161595806)


#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-11-08 at 12 11 54 pm" src="https://user-images.githubusercontent.com/32720508/48196104-c9a9ba80-e351-11e8-9654-708c6c532382.png">


#### Questions:
N/A
